src/eval.jl

```diff
@@ -1,6 +1,7 @@
 using CodeTools, LNR, Media
 import CodeTools: getthing
 
+preprocess(x) = x
```

Comment:
I mean that sounds find to me but I'm not sure if it makes a difference? 

I was planning on using this personally to replace this with
```
preprocess(text) = *("import ChainRecursive\nChainRecursive.@chain begin\n", text, "\nend")
```
but
```
myprocess(text) = *("import ChainRecursive\nChainRecursive.@chain begin\n", text, "\nend")
preprocess = myprocess
```
would work just as well